submissions/rafiatAminu/JavaScript/cond.js

```diff
@@ -0,0 +1,17 @@
+let a = 0;
+let b= 7;
+let c= 123;
+let d= (c-b);
+
+for(a=0; a<d;) {
```

Comment:
The `for()` take 3 arguments, two given. Also, kindly format this block of code to make it readable